.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 /.jwtsecret
 /example/.jwtsecret
```

Comment:
Add that dist folder to the `.gitignore` too so it doesn't pop up again if you build it again.